pipeline/errors/error_redirect.go

```diff
@@ -66,3 +68,19 @@ func (a *ErrorRedirect) Config(config json.RawMessage) (*ErrorRedirectConfig, er
 func (a *ErrorRedirect) GetID() string {
 	return "redirect"
 }
+
+func (a *ErrorRedirect) RedirectUrl(r *http.Request, c *ErrorRedirectConfig) string {
+	if c.UrlParam == "" {
+		return c.To
+	}
+
+	url, err := url.Parse(c.To)
```

Comment:
It would be better to not shadow the imported `url` package. a) the package can be used later on and b) it is clear that you use a variable and not the package.
Having very short variable names is common for short lived variables
```suggestion
	u, err := url.Parse(c.To)
```